.github/workflows/setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -ex
+
+add-apt-repository ppa:git-core/ppa
```

Comment:
4.32-latest fails in a way that the DataProvider fix in #93 won't fix. I think we still need a newer version of hg:
```bash
UBUNTU_VERSION=$(lsb_release -r -s)
if [ $(echo $UBUNTU_VERSION | cut -c1-2) -lt "18" ]; then
  add-apt-repository ppa:mercurial-ppa/releases
fi
```
(from `.travis.sh`)

Reply:
Dang, I guess the HHVM docker images must be running an old version of Ubuntu still.

Reply:
Yikes, 14.04 https://hub.docker.com/r/hhvm/hhvm